x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/pattern/index_check_flyout/index.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IlmExplainLifecycleLifecycleExplain } from '@elastic/elasticsearch/lib/api/types';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiSpacer,
+  EuiTab,
+  EuiTabs,
+  EuiText,
+  EuiTitle,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import moment from 'moment';
+import { useIndicesCheckContext } from '../../../contexts/indices_check_context';
+
+import { EMPTY_STAT, getDocsCount, getSizeInBytes } from '../../../helpers';
+import { MeteringStatsIndex, PatternRollup } from '../../../types';
+import { useDataQualityContext } from '../../data_quality_context';
+import { IndexProperties } from '../../index_properties';
+import { CHECK_NOW } from '../../summary_table/translations';
+import { getIlmPhase } from '../helpers';
+import { IndexResultBadge } from '../../index_result_badge';
+import { HISTORY, LATEST_CHECK } from '../translations';
+import { useCurrentWindowWidth } from '../../../use_current_window_width';
+
+export interface Props {
+  ilmExplain: Record<string, IlmExplainLifecycleLifecycleExplain> | null;
+  indexName: string;
+  pattern: string;
+  patternRollup: PatternRollup | undefined;
+  stats: Record<string, MeteringStatsIndex> | null;
+  onClose: () => void;
+}
+
+const tabs = [
+  {
+    id: LATEST_CHECK,
+    name: LATEST_CHECK,
+  },
+  {
+    id: HISTORY,
+    name: HISTORY,
+    disabled: true,
```

Comment:
Consider hiding the `History` tab in the screenshot below, because it's "coming soon", but not implemented:

![history](https://github.com/user-attachments/assets/4a6c2bb9-474d-4528-ba5e-174ee67e906e)

Reply:
if I hide it will have to hide Latest Check as well, because tabline with a single tab is weird. That's why I presented this in last DQD meeting as is and checked with Dhru whether this looks ok. From what I remember this was approved. but maybe @dhru42 can clarify one more time?

Reply:
We shouldn't show anything we're not releasing in the UI. Main reason is anything greyed out means that the user doesn't have the right subscription to access it. So for users who are on the highest subscription (Enterprise), that shows they dont have access to it which is a point of confusion.

Reply:
Since this historical functionality + redesign is releasing publically in 8.16 (together), is there a reason why we're having this discussion?

@andrew-goldstein @kapral18 maybe I'm missing context here.

Reply:
@dhru42 thanks for the context, I didn't know about the greyed out == lack of subscription.

Why we are having this discussion is because once I merge this it will appear in serverless release a week later and is generally ready for deploy. And this is not under a feature flag because this is not a new functionality but rather a UI overhaul which is unfeasable to hide behind a flag.

Ok so should I just remove the history tab or remove the entire tabline?

Reply:
@dhru42 @andrew-goldstein removed the tabline it looks like this now
![image](https://github.com/user-attachments/assets/3e27e86b-09b9-428d-8a32-8a2172d1ad17)